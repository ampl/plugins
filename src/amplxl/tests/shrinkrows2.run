reset;

param n := 2;
param m := 3;

set A := 1..n;
set B := 1..m;

param C{A,B};
param temp default n * m;

for {i in A, j in B}{
	let C[i,j] := temp;
	let temp := temp - 1;
}

table table2 OUT "amplxl" "shrinkrows2.xlsx":
	[A, B], C;

write table table2;

reset;

param n := 2;
param m := 3;

set A := 1..n;
set B := 1..m;

param C{A,B};

table table2 IN "amplxl" "shrinkrows2.xlsx":
	[A, B], C;

read table table2;

display C;

