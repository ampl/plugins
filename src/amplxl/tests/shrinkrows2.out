C :=
1 1   6
1 2   5
1 3   4
2 1   3
2 2   2
2 3   1
;

